lib/assignment_manager.rb

```diff
@@ -104,6 +105,14 @@ def set_article_from_database
     @article = exact_title_matches.first
   end
 
+  def check_wiki_edu_discouraged_article
+    category = Category.where('name = ? AND article_titles LIKE ?',
```

Comment:
I think the better way to do this is to find the category first, and then check whether the title is included in the catetgory's array of titles. Using LIKE might result in mismatches, such as blocking the assignment of "Animal" when "Animal rights" is on the blocklist.